app/controllers/parent/kids_controller.rb

```diff
@@ -0,0 +1,37 @@
+class Parent::KidsController < ApplicationController
+
+  def index
+    @kids = Kid.all
```

Comment:
We only want to view this parent's kids. This would show us all the kids in our DB.

`Kid.find_by_parent_id(@parent.user_id)` - not exactly that, but that idea. 😸